src/test/java/services/ParseServiceTest.java

```diff
@@ -0,0 +1,76 @@
+package services;
+
+import static org.junit.Assert.assertEquals;
+
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+import model.Fruit;
+import model.FruitRecord;
+import org.junit.Test;
+import services.impl.ParseServiceImpl;
+
+public class ParseServiceTest {
+    private static final ParseService parser = new ParseServiceImpl();
+    private static final String validData = "type,fruit,quantity"
+            + System.lineSeparator()
+            + "b,banana,20"
+            + System.lineSeparator()
+            + "b,apple,100"
+            + System.lineSeparator();
+    private static final FruitRecord bananaRecord
+            = new FruitRecord('b', "banana", 20);
+    private static final FruitRecord appleRecord
+            = new FruitRecord('b', "apple", 100);
+    private static final Fruit banana
+            = new Fruit("banana", 20);
+    private static final Fruit apple
+            = new Fruit("apple", 100);
+
+    @Test(expected = RuntimeException.class)
+    public void parseFromCsv_nullString_Exception() {
```

Comment:
```suggestion
    public void parseFromCsv_nullString_notOk() {
```
Let's rename methods according the rule from readme
<img width="857" alt="Screenshot 2022-02-23 at 13 06 19" src="https://user-images.githubusercontent.com/29378916/155307601-3624695f-3745-4e3e-8f3a-a62753e6ec79.png">